command/ca/provisionerbeta/provisioner.go

```diff
@@ -238,4 +241,44 @@ By default it will accept any SAN in the CSR.`,
 with the same instance will be accepted. By default only the first request
 will be accepted.`,
 	}
+
+	// Nebula provisioner flags
+	nebulaRootFlag = cli.StringFlag{
+		Name: "nebula-root",
+		Usage: `Root certificate (chain) <file> used to validate the signature on Nebula
+provisioning tokens.`,
+	}
 )
+
+func readNebulaRoots(rootFile string) ([][]byte, error) {
+	b, err := utils.ReadFile(rootFile)
+	if err != nil {
+		return nil, err
+	}
+
+	var crt *cert.NebulaCertificate
+	var certs []*cert.NebulaCertificate
+	for len(b) > 0 {
+		crt, b, err = cert.UnmarshalNebulaCertificateFromPEM(b)
```

Comment:
Maybe import `cert` as `nebula` here, like in `step-ca`, for clarity and consistency.

Reply:
Fixed with bf0a16f